### PR TITLE
backoffice-ui: add loanDetails page

### PR DESCRIPTION
**Pull request by zzacharo**
addresses #22

### CODE REVIEW

File: invenio_app_ils/config.py [hunk 1]
```diff
@@ -174,15 +174,15 @@ def _(x):
 # TODO: uncomment when this PR:
 # https://github.com/inveniosoftware/invenio-app/pull/32
 # is merged and released and if u want to work on UI
-# APP_DEFAULT_SECURE_HEADERS = {"content_security_policy": {}}
+APP_DEFAULT_SECURE_HEADERS = {"content_security_policy": {}}
```
Comment (ntarocco) [inline invenio_app_ils/config.py]: * Please remove all this and the TODO
* Increase `invenio-app` to `1.0.3`

File: invenio_app_ils/config.py [hunk 2]
```diff
@@ -174,15 +174,15 @@ def _(x):
...
 # OAI-PMH
 # =======
 OAISERVER_ID_PREFIX = "oai:invenio_app_ils.com:"
 
 ###############################################################################
 # Debug
 ###############################################################################
-DEBUG = True
+DEBUG = False
```
Comment (ntarocco) [inline invenio_app_ils/config.py]: revert

File: invenio_app_ils/templates/invenio_app_ils/backoffice.html [hunk 3]
```diff
@@ -1,30 +1,25 @@
-# -*- coding: utf-8 -*-
-#
-# Copyright (C) 2018 CERN.
-#
-# invenio-app-ils is free software; you can redistribute it and/or modify it
-# under the terms of the MIT License; see LICENSE file for more details.
+{# -*- coding: utf-8 -*-
```
Comment (ntarocco) [inline invenio_app_ils/templates/invenio_app_ils/backoffice.html:1]: What's happening here? :)

File: invenio_app_ils/ui/backoffice/src/index.scss [hunk 4]
```diff
@@ -0,0 +1,14 @@
+body {
```
Comment (ntarocco) [inline invenio_app_ils/ui/backoffice/src/index.scss:1]: https://emotion.sh/ 😎

File: invenio_app_ils/ui/backoffice/src/pages/LoanDetails/state/api.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+
+const loansBaseUrl = 'https://localhost:5000/api/circulation/loans';
+
+// In development create a personal token to be able to perform requests
+const dev_jwt_token =
+  'S9iy1CEI9u52gj10b2k1MzeCefNYcneYGI4ABtAfLAP9ykDpxZHsXtXBKRJG';
```
Comment (ntarocco) [inline invenio_app_ils/ui/backoffice/src/pages/LoanDetails/state/api.js]: remove

**Review by ntarocco — APPROVED** [review]

File: invenio_app_ils/templates/invenio_app_ils/backoffice.html (hunk 3 above)
Comment (zzacharo) [reply invenio_app_ils/templates/invenio_app_ils/backoffice.html:1]: without these brackets in the start it was rendered in the html ;) Now it's recognized as a comment.

File: invenio_app_ils/ui/backoffice/src/index.scss (hunk 4 above)
Comment (zzacharo) [reply invenio_app_ils/ui/backoffice/src/index.scss:1]: this is coming from the bootstrapping of the project. Didn't use any styling library yet.

**Comment by zzacharo** [comment]
@ntarocco I updated the code with the changes.

File: invenio_app_ils/config.py (hunk 2 above)
Comment (topless) [reply invenio_app_ils/config.py]: Food for thought, we can have the debug value deriving from an env value, so its `False` when prod but `True` when dev.

Comment (zzacharo) [reply invenio_app_ils/config.py]: Until now in prod we have a `.cfg` file that overrides all these.

**Review by topless — APPROVED** [review]

**Pull request merged**